host-configs/LLNL/lassen-clang@upstream.cmake

```diff
@@ -54,7 +54,7 @@ set(CMAKE_CUDA_FLAGS_RELWITHDEBINFO "-g -lineinfo ${CMAKE_CUDA_FLAGS_RELEASE}" C
 set(CMAKE_CUDA_FLAGS_DEBUG "-g -G -O0 -Xcompiler -O0" CACHE STRING "")
 
 # Uncomment this line to make nvcc output register usage for each kernel.
-# set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --resource-usage" CACHE STRING "" FORCE)
+ set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} --resource-usage" CACHE STRING "" FORCE)
```

Comment:
Maybe comment this out again?

Reply:
I kind of get sick of uncommenting it. Is there an issue with just leaving it on?

Reply:
It spits out a lot of output at build time that can make error messages hard to find. Most of the time I'm building with nvcc I don't want to see that info.